app/src/main/res/values-sw600dp-land/dimens.xml

```diff
@@ -420,4 +420,14 @@
   <dimen name="revision_card_fragment_layout_text_margin_top">32dp</dimen>
   <dimen name="revision_card_fragment_layout_button_margin_top">24dp</dimen>
 
+  <!--  AdministratorControlDownloadPermissionsView-->
```

Comment:
Please make sure spacing Is correct in comments.  e.g. `<!-- FAQ Single -->`
Ditto for others.